my-app/src/Components/cartItem/index.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export class CartItem extends React.Component {
+    render() {
+        return(
+            <div className="cart-item">
+                <span>{this.props.peak} |</span>    
+                <span> {this.props.county} |</span>    
+                <span> {this.props.cottage}</span>    
```

Comment:
poti pune un if aici: this.props.cottage ? this.props.cottage : "No cottage!" ;)